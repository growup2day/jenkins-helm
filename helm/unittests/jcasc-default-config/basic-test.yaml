templates:
  - config-jcasc.yaml
tests:
  - it: should include basic config in default config scripts when configured
    set:
      domain: dummy.domain
      JCasC:
        defaultConfig: true
    asserts:
      - matchSnapshot:
          path: data.[jcasc-default-config-basic.yaml]
  - it: should reflect configured values
    set:
      domain: dummy.domain
      JCasC:
        defaultConfig: true
        adminAddress: "test <test@test.com>"
        master:
          mode: "EXCLUSIVE"
          numExecutors: 3
    asserts:
      - matchSnapshot:
          path: data.[jcasc-default-config-basic.yaml]
