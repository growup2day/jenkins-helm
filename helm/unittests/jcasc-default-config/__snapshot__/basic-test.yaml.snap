should include basic config in default config scripts when configured:
  1: |
    |-
      jenkins:
        # Configure System > Maven Project Configuration
        mode: "NORMAL"
        numExecutors: 0
      unclassified:
        # Configure System > Jenkins Location
        location:
          url: "https://RELEASE-NAME-jenkins-NAMESPACE.dummy.domain/"
          adminAddress: ""
should reflect configured values:
  1: |
    |-
      jenkins:
        # Configure System > Maven Project Configuration
        mode: "EXCLUSIVE"
        numExecutors: 3
      unclassified:
        # Configure System > Jenkins Location
        location:
          url: "https://RELEASE-NAME-jenkins-NAMESPACE.dummy.domain/"
          adminAddress: "test <test@test.com>"
